package.json

```diff
@@ -48,11 +48,11 @@
   },
   "peerDependencies": {
     "@nomicfoundation/hardhat-ethers": "^3.0.0",
-    "@nomicfoundation/hardhat-verify": "^2.0.0",
+    "@nomicfoundation/hardhat-verify": "^2.0.13",
     "@openzeppelin/hardhat-upgrades": "^3.1.1",
     "@openzeppelin/upgrades-core": "^1.27.1",
     "@types/mocha": "^9.1.0",
-    "ethers": "^6.13.4",
+    "ethers": "6.13.5",
```

Comment:
Is it possible to use range?
Because there is different ethers version in different repositories.